.rubocop_todo.yml

```diff
@@ -1,43 +1,35 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2018-05-05 20:09:23 +0200 using RuboCop version 0.54.0.
+# on 2024-01-31 08:55:31 UTC using RuboCop version 1.23.0.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-require: rubocop-minitest
+# Offense count: 1
+# Cop supports --auto-correct.
+Lint/RedundantCopDisableDirective:
+  Exclude:
+    - 'app/models/monitoring_result.rb'
```

Comment:
Perhaps just drop the disable statement in the file?

Reply:
If you are talking about the `# rubocop:disable` statement , then I can't see it in the file.

Reply:
You can use `--auto-correct --only Lint/RedundantCopDisableDirective`.